pkg/reconciler/tenancy/bootstrap/bootstrap_controller.go

```diff
@@ -69,8 +69,9 @@ func NewController(
 		syncChecks: []cache.InformerSynced{
 			workspaceInformer.Informer().HasSynced,
 		},
-		workspaceType: workspaceType,
-		bootstrap:     bootstrap,
+		workspaceType:   workspaceType,
+		initializerPath: initializerPath,
```

Comment:
we talked about this. IMO wrong direction. But we agreed to explore our direction in a doc.

Reply:
Or in other words: this controller should have one and only one initializer path, namely `root`, because it is only used in ClusterWorkspaceTypes in `root`. This initializer is not an API to be used by anybody else than the core system.